### ISSUE TITLE
How to make preview more compact?

### ISSUE THREAD

**Issue by RichardJECooke** [asciidoctor/asciidoctor-vscode#426]
In preferences - asciidoc - I changed line height and font size - but my preview doesn't change at all. How do I make the preview more compact please?

**Comment by g0t4** [comment]
I have the same problem, it seems like settings for the extension aren't applied.

**Comment by g0t4** [comment]
Ok, I found my issue, make sure this is unchecked:

![CleanShot 2021-08-05 at 15 43 09](https://user-images.githubusercontent.com/180189/128411273-71e8d088-b3e3-4ad4-90e7-6f9545089829.png)

Even still changes aren't reflected immediately, I had to close / reopen for new styles to apply. I have a feeling I need to RTFM :)... I'll let you know what I find.

**Comment by RichardJECooke** [comment]
DIsabling that box and restarting changes the preview colors and font, but doesn't change the sizing at all, so I changed it back.

I think this must be a bug

**Comment by g0t4** [comment]
I agree it is buggy, I wish I had time to dig into it. It seems as if almost all settings are ineffectual (with regards to the appearance of the live preview) which really is too bad. Syntax highlighting also doesn't seem to work. The shortcut keys (on a mac) to show the preview also don't work. 

Time to look for an alternative rendering (perhaps a CLI call with all the settings I want and generate html that I open in a browser with live reload)

**Comment by g0t4** [comment]
This all said the builtin editing experience rocks, just the live preview has issues.

**Comment by danyill** [comment]
Thanks for filing this issue. 

Currently the `fontFamily`, `fontSize` and `lineHeight` options in the user settings all do nothing. I'm sorry for the bother, we should either fix or remove these options.

An option for making the live preview more compact would be to define your own CSS.

I had a 5-minute hunt for a CSS for handheld devices and found:

https://alistapart.com/article/pocket/

I downloaded https://alistapart.github.io/code-samples/pocket/handheld.css locally and then set my options as shown below:

![image](https://user-images.githubusercontent.com/674783/135695659-e50feaf9-6614-45bd-ae38-09f274203afa.png)

This did a reasonable job and allowed quite a readable view.

![image](https://user-images.githubusercontent.com/674783/135695779-c3a6a858-bfe2-4e97-8b18-d90f87343ffc.png)

I find the built-in stylesheet or the default Asciidoctor stylesheet fine to work with. Can you be more specific about what you would like in a preview? Are you using large displays or looking for a very compact layout?

**Comment by RichardJECooke** [comment]
Thanks, I'll try the CSS.

> Are you using large displays or looking for a very compact layout?
I want a more compact layout without the huge vertical whitespace. I use VS Code with Asciidoctor to keep all my notes on all subjects in a folder. And when I try read them just a simple bullet list with a dozen points takes up a screen. This is crazy - I want to see tons of info on one screen for quick reference. I really hope you can fix the line height spacing so I can set it to 1.

**Comment by g0t4** [comment]
Thanks!

**Comment by Mogztter** [comment]
Fixed in https://github.com/asciidoctor/asciidoctor-vscode/pull/530
